src/KristofferStrube.ActivityStreams/ActivityTypes/IntransitiveActivity.cs

```diff
@@ -3,55 +3,13 @@
-
-    /// <summary>
-    /// Describes an object of any kind. The Object type serves as the base type for most of the other kinds of objects defined in the Activity Vocabulary, including other Core types such as Activity, IntransitiveActivity, Collection and OrderedCollection.
-    /// </summary>
-    [JsonPropertyName("object")]
-    [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingDefault)]
-    [JsonConverter(typeof(OneOrMultipleConverter<IObjectOrLink>))]
-    public IEnumerable<IObjectOrLink>? Object { get; set; }
-
-    /// <summary>
-    /// Describes the indirect object, or target, of the activity. The precise meaning of the target is largely dependent on the type of action being described but will often be the object of the English preposition "to". For instance, in the activity "John added a movie to his wishlist", the target of the activity is John's wishlist. An activity can have more than one target.
-    /// </summary>
-    [JsonPropertyName("target")]
-    [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingDefault)]
-    [JsonConverter(typeof(OneOrMultipleConverter<IObjectOrLink>))]
-    public IEnumerable<IObjectOrLink>? Target { get; set; }
-
-    /// <summary>
-    /// Describes the result of the activity. For instance, if a particular action results in the creation of a new resource, the result property can be used to describe that new resource.
-    /// </summary>
-    [JsonPropertyName("result")]
-    [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingDefault)]
-    [JsonConverter(typeof(OneOrMultipleConverter<IObjectOrLink>))]
-    public IEnumerable<IObjectOrLink>? Result { get; set; }
-
-    /// <summary>
-    /// Describes an indirect object of the activity from which the activity is directed. The precise meaning of the origin is the object of the English preposition "from". For instance, in the activity "John moved an item to List B from List A", the origin of the activity is "List A".
-    /// </summary>
-    [JsonPropertyName("origin")]
-    [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingDefault)]
-    [JsonConverter(typeof(OneOrMultipleConverter<IObjectOrLink>))]
-    public IEnumerable<IObjectOrLink>? Origin { get; set; }
-
-    /// <summary>
-    /// Identifies one or more objects used (or to be used) in the completion of an Activity.
-    /// </summary>
-    [JsonPropertyName("instrument")]
-    [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingDefault)]
-    [JsonConverter(typeof(OneOrMultipleConverter<IObjectOrLink>))]
-    public IEnumerable<IObjectOrLink>? Instrument { get; set; }
+  /// <summary>
+  /// Instances of IntransitiveActivity are a subtype of Activity representing intransitive actions. The object property is therefore inappropriate for these activities. 
+  /// </summary>
+  [Obsolete("The object property is inappropriate for intransitive activities.", true)]
+  private new IEnumerable<IObjectOrLink>? Object { get => base.Object; set => base.Object = value; }
```

Comment:
Could you change this to use tabs instead of spaces for consistency?

Reply:
I've done that now (through the editor in GitHub).
Usually my code editor will automatically adapt to whatever the file has. Not sure why it didn't this time.